specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/2015-06-15/Network.json

```diff
@@ -0,0 +1,283 @@
+                },
+                "loadBalancerMuxHealth": {
+                    "description": "Health for all LB MUXes as reported by the NC.",
+                    "$ref": "#/definitions/AdminOverviewResourceHealth"
+                },
+                "virtualGatewayHealth": {
+                    "description": "Health for all gateways as reported by the NC.",
+                    "$ref": "#/definitions/AdminOverviewResourceHealth"
+                },
+                "publicIpAddressUsage": {
+                    "description": "Total Public IP usage as reported by the NC.",
+                    "$ref": "#/definitions/AdminOverviewResourceUsage"
+                },
+                "backendIpUsage": {
+                    "description": "Total Backend IP usage as reported by the NC.",
+                    "$ref": "#/definitions/AdminOverviewResourceUsage"
+                },
+                "macAddressUsage": {
+                    "description": "Total MAC address usage as reported by the NC.",
+                    "$ref": "#/definitions/AdminOverviewResourceUsage"
+                }
+            }
+        },
+        "AdminOverviewResourceHealth": {
+            "description": "Resource health information.",
+            "properties": {
+                "totalResourceCount": {
+                    "description": "Total number of resources.",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "healthyResourceCount": {
+                    "description": "Number of resources in the success state.",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "errorResourceCount": {
+                    "description": "Number of resources in the error state.",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "warningResourceCount": {
+                    "description": "Number of resources in the warning state.",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "healthUnknownCount": {
+                    "description": "Number of resources in an unknown state.",
+                    "type": "integer",
+                    "format": "int64"
+                }
+            }
+        },
+        "AdminOverviewResourceUsage": {
+            "description": "Resource usage information.",
+            "properties": {
+                "totalResourceCount": {
+                    "description": "Total number of resources.",
+                    "type": "integer",
+                    "format": "int64"
+                },
+                "inUseResourceCount": {
+                    "description": "Number of resources in use.",
+                    "type": "integer",
+                    "format": "int64"
+                }
+            }
+        }
+    },
```

Comment:
can you also check whether properties and nested properties of `AdminOverview` model are eligible to be marked as readonly?